frontend/src/modules/Survey/Components/Survey.tsx

```diff
@@ -24,30 +24,32 @@ export const Survey = () => {
 
   const [showError, setShowError] = useState(false)
   const [currStep, setCurrStep] = useState(0)
+  const defaultSurvey: SurveyForm = require('@core/Questions/DefaultSurvey.json')
+  const [survey, setSurvey] = useState<SurveyForm>(defaultSurvey)
+  // const numSpecialQuestions = 0 // don't think we need this anymore due to the api call
+  const totalSteps = survey.questions.length
+
   // If there are custom questions the below will be a network call perhaps
   // ^ here it is baby:
   useEffect(() => {
     const fetcher = async () => {
       axios.get(`${API_ROOT}${COURSE_API}/${surveyId}${SURVEY_API}`).then(
         (response: any) => {
-          setQuestions(response.data)
+          setSurvey(response.data)
         },
         (error: any) => {
           console.log(error)
         }
       )
     }
+    fetcher()
```

Comment:
this guy wasn't being called before, resulting in it defaulting to the old response format in DefaultSurvey